src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -31,7 +31,8 @@ const DomainList = (props) => {
     onPressBody,
     DefaultImage,
     isCommunity,
-    isBlockedSection
+    isBlockedSection,
+    isDomain
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isDomain` is added but it's not clear how it's used in this component. Make sure to add its usage or remove it if it's not necessary.



<!-- This is an auto-generated comment by OSS CodeRabbit -->